src/visGeometry/index.ts

```diff
@@ -1060,21 +1060,44 @@ class VisGeometry {
         }
     }
 
+    /**
+     * AGENT COLOR HANDLING (TODO: move to separate file)
+     * General notes about data being used to map color to agents:
+     * @property this.colorsData is an array of floats, each 4 floats is a color
+     *  `dataColorIndex` is always an index into the colorsData array, so it is a multiple of 4
+     * `colorId` is always an index into the numberOfColors, so it is a number between 0
```

Comment:
Nit: The phrase `index into the numberOfColors` seems wrong here, since it suggests that `numberOfColors` is an array?